lib/bullhorn.ex

```diff
@@ -1,4 +1,4 @@
-defmodule CopyCat do
+defmodule bullhorn do
```

Comment:
```suggestion
defmodule Bullhorn do
```

Reply:
Dang, good catch :+1: I used VS since my vim script seemed to be broken. I don't know how it missed this file with find & replace all :panic: